drone/values.yaml

```diff
@@ -0,0 +1,66 @@
+image:
+  registry: docker.io
+  org: drone
+  name: drone
+  tag: 0.7.3
+  pullPolicy: IfNotPresent
+
+service:
+  http:
+    externalPort: 80
+    internalPort: 8000
+  type: NodePort
+
+ingress:
+  enabled: true
+  # enable TLS via kube-lego
+  tls: false
+  hostname: drone2.srcd.host
+  annotations:
+    kubernetes.io/ingress.class: gce
+    kubernetes.io/ingress.global-static-ip-name: drone2-srcd-host
```

Comment:
This annotation value is quite specific for srcd. I guess at some point you will pass it as a parameter. I would comment make a note either here and/or README file